application/views/admin/token/massive_actions/_update.php

```diff
@@ -422,14 +422,6 @@ class="YesNoDatePicker form-control action_datepickerUpdateHiddenField"
 
 <?php App()->getClientScript()->registerScript("Tokens:MassActionUpdateView_Scripts", "
 
-   var bindBSSwitch = function(formGroup){
-       console.log(\"bindBSSwitch run on:\",formGroup);
-       //Script to update the completed settings
-       formGroup.find('.YesNoSwitch').on('switchChange.bootstrapSwitch', function(e, state){        
```

Comment:
Another piece of logic we need to test?

Reply:
my mistake.
This should not be removed. :(

Reply:
Alright, that's great, but I still might need testing instructions. :D For Tibor at least.